docs/wiki/Build-from-Source.md

```diff
@@ -62,7 +62,7 @@ make build
 `csp-adapter-symphony` has linting and auto formatting.
 
 | Language | Linter      | Autoformatter | Description |
-| :------- | :---- ----- | :------------ | :---------- |
+| :\------- | :---- ----- | :------------ | :---------- |
```

Comment:
is this intentional?

Reply:
some version of `mdformat` locally thought it wanted to do it, dont know why.